benchmark/compare.js

```diff
@@ -1,10 +1,12 @@
 var usage = 'node benchmark/compare.js ' +
             '<node-binary1> <node-binary2> ' +
-            '[--html] [--red|-r] [--green|-g]';
+            '[--html] [--red|-r] [--green|-g] ' +
+            '[-- <type> [testFilter]]';
 
 var show = 'both';
 var nodes = [];
 var html = false;
+var benchmarks;
```

Comment:
Clarified via irc that it is using the naming found in https://github.com/iojs/io.js/blob/v1.x/benchmark/common.js#L10-L37